app/assets/stylesheets/media.scss

```diff
@@ -101,8 +101,13 @@
     visibility: hidden;
   }
 
+  // Ensure fullscreen media is as large as possible.
   &.vjs-fullscreen {
-    height: auto;
+    audio,
+    video {
+      height: 100% !important;
```

Comment:
`!important` is being used here and above because we're overriding styles applied directly to the element.